preview/MsixCore/msixmgr/ApplyACLsProvider.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <string>
+#include <vector>
+
+namespace MsixCoreLib
+{
+    class ApplyACLsProvider
+    {
+    public:
+
+        ApplyACLsProvider() {};
+        ~ApplyACLsProvider() {};

```

Comment:
you don't need a class. just use a namespace if you have a flat function that does everything. 
My point about class is you could've chosen to integrate the loadlibrary/getprocaddress into the class itself similar to msixmodules.h